### PR TITLE
Update snippy to 4.0.0

### PR DESCRIPTION
**Pull request by tseemann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/snippy/3.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name }}
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  sha256: {{ hash }}
+  url: https://github.com/tseemann/{{ name }}/archive/v{{ version }}.tar.gz
+
+build:
+  number: 1
```
Comment (druvus) [inline recipes/snippy/3.2/meta.yaml]: Please bump this to 2.

**Comment by druvus** [comment]
I think you also want to wait for Bioperl (#10325) to be rebuilt before merging.

**Comment by abretaud** [comment]
I'll have to make a few other PR after #10325 to finish rebuilding all the bioperl packages, I hope it will be ok tomorrow

File: recipes/snippy/3.2/meta.yaml (hunk 1 above)
Comment (tseemann) [reply recipes/snippy/3.2/meta.yaml]: Do i have to manually increment this at every build *attempt* ?

**Comment by tseemann** [comment]
OK i will wait. I have another 8 packages to add/update that depend on Perl (and often Bioperl) too.

File: recipes/snippy/3.2/meta.yaml (hunk 1 above)
Comment (abretaud) [reply recipes/snippy/3.2/meta.yaml]: Hum, you only need to increment it once in the same pull request. It's only when the pull request is merged that the package with this build number is uploaded to anaconda.org

**Comment by abretaud** [comment]
bioperl should be ok now, if you bump the build number we'll see if your update builds fine

**Comment by tseemann** [comment]
Thanks @abretaud !  Trying now. Does bioperl also work on OSX now?

**Comment by tseemann** [comment]
```
Can't build /Users/distiller/project/recipes/snippy due to environment creation error:
Unsatisfiable dependencies for platform osx-64: {"vt[version='>=0.5772']"}
```
I forgot that my `vt` recipe fails on clang.  Skipping osx now.

**Pull request merged**